internal/parser/parser.go

```diff
@@ -1875,11 +1875,14 @@ func (p *Parser) parseErrorForMissingSemicolonAfter(node *ast.Node) {
 		return
 	}
 	// !!! The user alternatively might have misspelled or forgotten to add a space after a common keyword.
```

Comment:
Does `!!!` in comments mean a `TODO`? Should I remove it from this comment? :thinking: